src/store/index.js

```diff
@@ -2,15 +2,13 @@ import { createStore, applyMiddleware, compose } from "redux";
 import thunk from "redux-thunk";
 import { changeURLMiddleware } from "../middleware/changeURL";
 import rootReducer from "../reducers";
-import { loggingMiddleware } from "../middleware/logActions";
 import { keepScatterplotStateInSync } from "../middleware/scatterplot";
 
 const configureStore = (initialState) => {
   const middleware = [
     thunk,
     keepScatterplotStateInSync,
-    changeURLMiddleware,
-    // loggingMiddleware
```

Comment:
`loggingMiddleware` is very helpful during development and debugging. I think we've kept it commented out here for "easy access". Should we make it just part of the dev environment with something like 
```js
(process.env.NODE_ENV !== 'production' && loggingMiddleware)
```

Reply:
It's a bit too noisy for "normal" development work, but I'd be fine with it being under a different environment variable (e.g. `LOG_ALL_ACTIONS=1 auspice develop`). Not necessary for this PR tho.

Reply:
> please keep it there!

Dropped 1e1d441094951003f252f741cba5b876258b988f.